spyder/plugins/completion/languageserver/client.py

```diff
@@ -70,8 +69,8 @@ class LSPClient(QObject, LSPMethodProviderMixIn):
     sig_server_error = Signal(str)
 
     #: Signal to warn the user when either the transport layer or the
-    #  server are down
-    sig_lsp_down = Signal(str)
+    #  server went down
+    sig_went_down = Signal(str)
```

Comment:
Not related to this PR, but we might consider in the future documenting signals like
```python
sig_went_down = Signal(str)
"""Documentation of the signal.""""
```

This is what sphinx will parse